exercises/edward_brown/ex01_basics/include/Punctuation.h

```diff
@@ -0,0 +1,20 @@
+#ifndef EX1_PUNCTUATION
+#define EX1_PUNCTUATION
+#include <string>
+
+/** Determines if Candidate holds any of the following character values
+ *  `.,?'"!():`.
+ *
+ *  @param The character to test.
+ *  @return True if the Candidate is a punctuation mark, false if it is not.
+ */
+bool isPunctuation(char Candidate);
```

Comment:
Good point. You're right - I just did a quick search and found some quite verbose descriptions in headers in this case, so it looks like that's fine :+1: